lib/byron/src/Cardano/Wallet/Byron/Network.hs

```diff
@@ -378,6 +354,12 @@ mkWalletClient bp chainSyncQ =
         , getGenesisBlockHash
         } = bp
 
+    codecs :: MonadST m => ClientCodecs ByronBlock m
+    codecs = clientCodecs
+        (ByronCodecConfig (toEpochSlots getEpochLength))
+        ByronNodeToClientVersion1
```

Comment:
Aren't we using `Version2` ?

Reply:
Hm… I was going to say that this is another kind of version… but yes — we should probably use https://github.com/input-output-hk/ouroboros-network/blob/02a808741ef3fc8b244c4be221616e4fc92a8310/ouroboros-consensus-byron/src/Ouroboros/Consensus/Byron/Ledger/NetworkProtocolVersion.hs#L32

Reply:
Done